modules/tinymce/src/themes/mobile/main/ts/style/Styles.ts

```diff
@@ -7,13 +7,15 @@
 
 import { Fun } from '@ephox/katamari';
 
-const prefix = 'tinymce-mobile';
+const sPrefix = 'tinymce-mobile';
```

Comment:
Another variable name we maybe should rethink, since `sName` is normally used for agar steps?

Reply:
done